playbooks/ovirt-host-remove.yml

```diff
@@ -0,0 +1,37 @@
+- hosts: all
+  remote_user: root
+  vars:
+      hosted_engine_config_file: "/etc/ovirt-hosted-engine/hosted-engine.conf"
+
+  tasks:
+    - name: Checks hosted engine configuration file
+      stat:
+        path: "{{ hosted_engine_config_file }}"
+      register: conf_file
```

Comment:
We are checking existence of a file several times, so the playbook may be inconsistent if some other process creates/removes the file during execution. Wouldn't it be better to check the file existence only once and afterward use only the result?

Reply:
I thought _register_ assigns result of command (in this case stat) to the variable (conf_file). So, if I understand it correctly, the check should be done only once and then we work only with result of this check.

Reply:
Right, my mistake, thanks!